.github/workflows/docker-image-common.yml

```diff
@@ -86,7 +83,7 @@ jobs:
       - name: Set image tag for docs build
         id: set-image-tag
         run: |
-          echo "IMAGE_TAG=$(echo '${{ steps.meta.outputs.tags }}' | cut -d',' -f1)" >> $GITHUB_ENV
+          echo "IMAGE_TAG=$(echo '${{ steps.meta.outputs.tags }}' | head -n 1 | cut -d',' -f1)" >> $GITHUB_ENV
```

Comment:
I think the problem was that the delimiter in `steps.meta.outputs.tags` was newline instead of comma. I added `| head -n 1` to handle that. I'm not sure why I had the comma-handling bit (`cut ...`), so I kept that too.